src/lib/cli/command.js

```diff
@@ -603,7 +603,7 @@ export default function ( opts ) {
 	if ( _opts.format ) {
 		args.option(
 			'format',
-			'Render output in a particular format. Accepts "table" (default), "csv", and "json".'
+			'Render output in a particular format. Accepts “table” (default), “csv”, and “json”.'
```

Comment:
It looks like we need to add back `table` here as well; see https://github.com/Automattic/vip-cli/blame/00f33750140521dfada602abee9bd6faffa368e5/src/lib/cli/command.js#L604-L608

Reply:
The change was introduced in #2054 (https://github.com/Automattic/vip-cli/pull/2054/files#diff-0cb02012dbe53ce796952c6ca1bddd7fb9b09e4a86ca914b4240552409e84fdcL606-R606)